src/TBRunTestInBatches.js

```diff
@@ -0,0 +1,28 @@
+import partitionAll from "partition-all";
+
+export class TBBatchRunner {
```

Comment:
The files that export classes should probably be named the same as the class?

Eg `TBBatchRunner.js` exports `TBBatchRunner` instead of `TBRunTestInBatches.js`